logging-ktor-client/build.gradle.kts

```diff
@@ -0,0 +1,100 @@
+plugins {
+    kotlin("multiplatform")
+    id("kotlinx-atomicfu")
+    id("org.jmailen.kotlinter")
+    jacoco
+    id("org.jetbrains.dokka")
+    id("com.vanniktech.maven.publish")
+}
+
+apply(from = rootProject.file("gradle/jacoco.gradle.kts"))
+
+kotlin {
+    explicitApi()
+
+    jvm()
+    js().browser()
+    macosX64()
+    iosX64()
+    iosArm32()
+    iosArm64()
+
+    sourceSets {
+        val commonMain by getting {
+            dependencies {
+                api(project(":logging"))
+                api(ktor.client("core"))
+                api(ktor.client("logging"))
+            }
+        }
+
+        val commonTest by getting {
+            dependencies {
+                implementation(project(":logging-test"))
+                implementation(project(":test"))
+                implementation(kotlin("test-common"))
+                implementation(kotlin("test-annotations-common"))
+                implementation(ktor.client("mock"))
+            }
+        }
+
+        val jvmTest by getting {
+            dependencies {
+                implementation(kotlin("test-junit"))
+            }
+        }
+
+        val jsTest by getting {
+            dependencies {
+                implementation(kotlin("test-js"))
+            }
+        }
+
+        val appleMain by creating {
+            dependsOn(commonMain)
+        }
+
+        val appleTest by creating {
+            dependsOn(commonTest)
+            dependencies {
+                implementation(kotlinx.coroutines("core", "1.5.2-native-mt")) {
+                    version {
+                        strictly("1.5.2-native-mt")
+                    }
+                }
```

Comment:
Why is the multithreaded version of Coroutines needed?

Reply:
Ktor in Kotlin/Native [checks for it at runtime](https://github.com/ktorio/ktor/blob/main/ktor-client/ktor-client-core/posix/src/io/ktor/client/utils/CoroutineUtilsPosix.kt#L20), otherwise tests failed.

Reply:
Woah, had no idea. TIL